app/src/test/java/com/example/bottomnav/ParseTest.java

```diff
@@ -174,4 +173,25 @@ public void parseData() throws Exception {
         assertEquals("uint8_t", struct.get(4).type);
         assertEquals("uint8_t", struct.get(5).type);
     }
+
+    @Test
+    public void checkIDToStructMap() throws Exception {
+        Parse test = Parse.parseTextFile("parseData.h");
+        ArrayList<StructContents> struct1 = test.getAssociatedStruct("chargerstatus");
```

Comment:
Consider doing more of an end-to-end test - instead of providing the stylized input, can you pass in the CAN ID name as you'd get in actual usage?